ush/templates/FV3LAM_wflow.xml

```diff
@@ -369,11 +389,84 @@ MODULES_RUN_TASK_FP script.
     </dependency>
 
   </task>
+
+{%- if do_dacycle %}
+<!--
+************************************************************************
+************************************************************************
+-->
+  <task name="&ANAL_GSI_INPUT_TN;" cycledefs="forecast" maxtries="2">
+
+    &RSRV_DEFAULT;
+    &WALL_LIMIT_ANAL;
+    <!--&WALL_LIMIT_WHOLE;-->
+
+    <command>&LOAD_MODULES_RUN_TASK_FP; "&ANAL_GSI_INPUT_TN;" "&JOBSDIR;/JREGIONAL_RUN_ANAL"</command>
```

Comment:
Following on from my comment about the modulefiles, you can just put "anal_gsi_input" here in stead of `ANAL_GSI_INPUT_TN` to load the appropriate module.

Reply:
Yes, changed